lib/ansible/modules/cloud/docker/docker_network.py

```diff
@@ -598,6 +628,7 @@ def main():
     option_minimal_versions = dict(
         scope=dict(docker_py_version='2.6.0', docker_api_version='1.30'),
         attachable=dict(docker_py_version='2.0.0', docker_api_version='1.26'),
+        labels=dict(docker_py_version='1.10.0', docker_api_version='1.23'),
```

Comment:
Since we require docker-py 1.10.0 anyway, you can leave that part away.
```suggestion
        labels=dict(docker_api_version='1.23'),
```